src/createManager.js

```diff
@@ -72,7 +72,7 @@ const protoManager = {
 
   openMenu(openOptions) {
     if (this.isOpen) return;
-    openOptions = openOptions || {};
+    openOptions = openOptions || { focusMenu: true };
```

Comment:
Good one, didn't think of that. If the repo supports es6 destructuring & default parameters I could try something like this: 

`openMenu({ focusMenu = true }) {
    // etc.
  }
`

I'll have a look if that works.